src/js/events/SystemLogActions.js

```diff
@@ -39,17 +76,14 @@ const SystemLogActions = {
    * @param {String} [options.read_reverse] will read events in reverse order if set to true
    * @return {Symbol} subscriptionID to unsubscribe or resubscribe with
    */
-  subscribe(nodeID, options = {}) {
+  startTail(nodeID, options = {}) {
     let {subscriptionID, cursor, skip_prev} = options;
 
     // Unsubscribe if any open connection exists with the same ID
-    this.unsubscribe(subscriptionID);
+    this.stopTail(subscriptionID);
 
     const url = SystemLogUtil.getUrl(nodeID, options);
-    subscriptionID = subscriptionID || Symbol(url + Date.now());
-    const source = new EventSource(url, {
-      withCredentials: Boolean(CookieUtils.getUserMetadata())
-    });
+    subscriptionID = subscriptionID || Symbol.for(url);
```

Comment:
Thinking about it a bit more… when I am requesting the same stream again, I guess it would make sense to close any outgoing connection, if there is one open. I think you are right, that it is not needed